workshop/posting-errors-guidelines.md

```diff
@@ -189,25 +173,28 @@ Either method will format your post like below:
 
 If all looks set, click the green arrow to add it to the thread!
 
-**Slack Tip:** if you see a mistake in your post you missed, you can click on the three dots in the corner of your Slack post and choose `Edit message` and fix the mistake:
+**Slack Tip:** if you see a mistake in your post you missed, you can click on the three dots in the corner of your Slack post.
```

Comment:
Have we seen people have trouble with this? I feel like the one screenshot was sufficient.

Reply:
Can you clarify this comment? Do you mean get rid of `<img src = "screenshots/slack-more-actions.png" width = "200">` on the next line?

Reply:
Yes, that is what I meant. It was also a genuine question, because I don't know if people have trouble knowing what we mean by/finding the three dots.

Reply:
> because I don't know if people have trouble knowing what we mean by/finding the three dots.

I too have no idea...!